src/System.Security.Cryptography.X509Certificates/tests/CtorTests.cs

```diff
@@ -312,12 +312,23 @@ public static void TestByteArrayConstructor_SerializedCert_Unix()
         [Fact]
         public static void TestNullConstructorArguments()
         {
-            Assert.Throws<ArgumentException>(() => new X509Certificate2((byte[])null, (String)null));
-            Assert.Throws<ArgumentException>(() => new X509Certificate2(new byte[0], (String)null));
-            Assert.Throws<ArgumentException>(() => new X509Certificate2((byte[])null, (String)null, X509KeyStorageFlags.DefaultKeySet));
-            Assert.Throws<ArgumentException>(() => new X509Certificate2(new byte[0], (String)null, X509KeyStorageFlags.DefaultKeySet));
-#if netstandard17
-            Assert.Throws<ArgumentNullException>(() => new X509Certificate2((X509Certificate2)null));
+            Assert.Throws<ArgumentNullException>(() => new X509Certificate2((string)null));
+            Assert.Throws<ArgumentException>(() => new X509Certificate2(IntPtr.Zero));
+            Assert.Throws<ArgumentException>(() => new X509Certificate2((byte[])null, (string)null));
+            using (new X509Certificate2(TestData.MsCertificate, (string)null)) { }
```

Comment:
Perhaps the non-throws ones should be in a different logical block, like in the netcoreapp11 section?